### DISCUSSION TITLE
Question : CSD theme

### DISCUSSION THREAD

**Discussion by mt190502** [rafaelmardojai/firefox-gnome-theme#228] [Q&A] [answered]
Hello 

I installed Firefox gnome theme. And I found it quite successful. Is it possible for the bar above to be the same as the custom theme in the system? (e.q Epiphany)

**Comment by gitSaptarshi** [comment, marked as answer]
1. To make the header bar look like you wanted go to _Customize_ option in firefox and add or remove items to the bar.
2. To make single-tab autohide go to _about:config_ then create a new boolean value `gnomeTheme.hideSingleTab` and set it to **true**. But before doing that move the new tab button to the header bar.
3. To drag the window from header bar button (like GNOME) create a new boolean value named `gnomeTheme.dragWindowHeaderbarButtons` from _about:config_ and set it to **true**.

It is advisable (from the developer) and also from my perspective **not to enable option 3**. This fuction is quite buggy and can crash firefox.
